virtualizarr/manifests/manifest.py

```diff
@@ -1,11 +1,11 @@
 import re
 from collections.abc import ItemsView, Iterable, Iterator, KeysView, ValuesView
 from pathlib import PosixPath
-from typing import Any, Callable, NewType, Tuple, TypedDict, cast
+from typing import Any, Callable, NewType, TypedDict, cast
```

Comment:
Import `Callable` from `collections.abc`. It was added there in py3.9 and importing it from `typing` was deprecated.

Reply:
thanks, done in [`3c2cba4` (#836)](https://github.com/zarr-developers/VirtualiZarr/pull/836/commits/3c2cba4e87fa8812982ed5c7f6b4dbd35b3997af)